### PR TITLE
feat: Added Landing Page

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Landing page with various categories to select added.

### CODE REVIEW

**Review by rahul0396 — APPROVED** [review]
Code is good and optimized as well as clean. Good to merge

File: Assets/SVG/Ninga.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M325.4 289.2L224 390.6 122.6 289.2C54 295.3 0 352.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-70.2-54-127.1-122.6-133.2zM32 192c27.3 0 51.8-11.5 69.2-29.7 15.1 53.9 64 93.7 122.8 93.7 70.7 0 128-57.3 128-128S294.7 0 224 0c-50.4 0-93.6 29.4-114.5 71.8C92.1 47.8 64 32 32 32c0 33.4 17.1 62.8 43.1 80-26 17.2-43.1 46.6-43.1 80zm144-96h96c17.7 0 32 14.3 32 32H144c0-17.7 14.3-32 32-32z"/></svg>
+<!--
+Font Awesome Free 5.2.0 by @fontawesome - https://fontawesome.com
+License - https://fontawesome.com/license (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+-->
```
Comment (shuklahimanshu81) [inline Assets/SVG/Ninga.svg:5]: Remove unnecessary comments

Comment (Hamdrive) [inline Assets/SVG/Ninga.svg:5]: Clean up unwanted code comments

File: style.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    --bg-color: #1f2937;
+}
+
+.ham-menu {
+    display: none;
+}
+
+.navbar {
+    background-color: var( --bg-color);
+    color: white;
```
Comment (AnuragSahu11) [inline style.css:21]: use color variable

**Review by Hamdrive — COMMENTED** [review]
Good work done, code looks neat!
Have suggested one cleanup of code comments, else looks good and ready to merge!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                <li>
+                    <ion-icon name="logo-twitter"></ion-icon><a class="footer-link" href="https://twitter.com/maruthithedev" target="_blank">@maruthithedev</a></li>
+            </ul>
+
+        </div>
+
+    </div>
+
+
+
```
Comment (shuklahimanshu81) [inline index.html:174]: Remove unnecessary spaces

**Review by AnuragSahu11 — APPROVED** [review]
Good looks good you can merge, suggested a change

**Review by shuklahimanshu81 — APPROVED** [review]
Code structure is super clean. Just remove extra spaces and comments. Everything else is fine. Good to go for merge.

File: Assets/SVG/Ninga.svg (hunk 1 above)
Comment (MaruthiPothuganti) [reply Assets/SVG/Ninga.svg:5]: Oh. Thanks for pointing it out. Will remove the comments.

Comment (MaruthiPothuganti) [reply Assets/SVG/Ninga.svg:5]: Thankyou, will do that

File: style.css (hunk 2 above)
Comment (MaruthiPothuganti) [reply style.css:21]: Thankyou for the suggestion. Made the changes

**Pull request merged**